README.md

```diff
@@ -1 +1,21 @@
-# calypso-iframe
\ No newline at end of file
+```        _                             _  __                          
+  ___ __ _| |_   _ _ __  ___  ___       (_)/ _|_ __ __ _ _ __ ___   ___ 
+ / __/ _` | | | | | '_ \/ __|/ _ \ _____| | |_| '__/ _` | '_ ` _ \ / _ \
+| (_| (_| | | |_| | |_) \__ \ (_) |_____| |  _| | | (_| | | | | | |  __/
+ \___\__,_|_|\__, | .__/|___/\___/      |_|_| |_|  \__,_|_| |_| |_|\___|
+             |___/|_|                                                   
+```
+
+This script will load `your.iframe.url` provided a DOM node with the following attributes is present:
+
+```
+<div id="calypso" data-url="https://your.iframe.url"></div>
```

Comment:
If you want to make it generic, I would pass the `id` of the DOM node as well as attribute
```html
<div id="your-div-id" data-url="https://your.iframe.url" data-ref="your-div-id"></div>
```

Reply:
I would probably rename the repository to make it generic as well such as "lazyload-iframe"

Reply:
I do think that's a great idea to make it generic 👌

Reply:
> I would probably rename the repository to make it generic as well such as "lazyload-iframe"

Hmm, 🤔 it's a good idea to make it generic but i'm not sure `lazyload-iframe` would be the best name, as it's not really lazyloaded but only async loaded, and the purpose of this repo is more to to communicate and get the iframe's height... 

Any idea ?

Reply:
`async-responsive-iframe`?

Reply:
> If you want to make it generic, I would pass the `id` of the DOM node as well as attribute
> 
> ```
> <div id="your-div-id" data-url="https://your.iframe.url" data-ref="your-div-id"></div>
> ```

I'm still not sure what you mean here ? We need to now at least a unique attribute of the DOM node we are targeting. It can be the `data-ref` attribute if we want to let the user able to target this node with an #id for it's own purposes, but this attribute value then needs to be the same for everyone. In order to make the repo more generic it could be `data-ref="async-responsive-iframe"`

Reply:
OK for the sake of simplicity:
```html
<div id="async-responsive-iframe" data-url="https://your.iframe.url"></div>
```